.github/pull_request_template.md

```diff
@@ -0,0 +1,49 @@
+## Purpose
+<!-- Describe the problems, issues, or needs driving this feature/fix and include links to related issues in the following format: Resolves issue1, issue2, etc. -->

```

Comment:
It's better to check with a higher-level person to see if converting this to comments is acceptable, as this has been the behavior since the repository was created.

Reply:
@janakamarasena @pamodaaw

Reply:
Ideally this information should only be visible to the PR author in edit mode and provides no value to anyone who reads the PR description. In the current way, it is very common for PR authors to mix the actual content with these "instructions", so everything is inside blockquotes. On the other hand, removing these placeholder text per each PR is tedious. 

We did the same change in identity-apps repo and it's saving us some time when creating pull requests, so thought to do the same change here. anyways, will check with the team and get back.